### PR TITLE
Remove UnsupportedTypeException

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java [hunk 1]
```diff
@@ -280,7 +281,7 @@ private DataType fromCassandraDataType(
       case BLOB:
         return DataType.BLOB;
       default:
-        throw new UnsupportedTypeException(
+        throw new ExecutionException(
```
Comment (feeblefakie) [inline core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java:284]: Any particular reason for throwing a checked exception here?
Looks like it's a programs' error or user's error so should they be an assertion error or unchecked exception?

File: core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java [hunk 2]
```diff
@@ -307,7 +308,7 @@ private DataType fromCassandraDataType(
       case BLOB:
         return com.datastax.driver.core.DataType.blob();
       default:
-        throw new UnsupportedOperationException(String.format("%s is not yet supported", dataType));
+        throw new AssertionError();
```
Comment (feeblefakie) [inline core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java:311]: `dataType` is mostly given from users?
If so, should it be an unchecked exception?

**Review by feeblefakie — COMMENTED** [review]
Overall looking good! Thank you!
Left some questions for clarification.

File: core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java (hunk 1 above)
Comment (feeblefakie) [reply core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java:284]: Thank you for the discussion! I understand the point so let's leave it as it is.

File: core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java (hunk 2 above)
Comment (feeblefakie) [reply core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java:311]: I understand the point. Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by thongdk8 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**